secure/storage/src/tests/suite.rs

```diff
@@ -191,8 +191,9 @@ fn test_create_key_pair_and_perform_rotations(storage: &mut dyn Storage) {
         let new_public_key = storage.rotate_key(CRYPTO_NAME).unwrap();
         let new_private_key = storage.export_private_key(CRYPTO_NAME).unwrap();
 
-        let exported_key =
-            storage.export_private_key_for_version(CRYPTO_NAME, public_key).unwrap();
+        let exported_key = storage
```

Comment:
I'm surprised the linter didn't catch this?

Reply:
what though?

Reply:
Ooh sorry -- I was confused about why this line changed wrapping? It seemed to be 2 lines before but then changed to 3 😕